Makefile

```diff
@@ -53,13 +53,14 @@ clean_hard: clean
 # Compiles sass to css
 css:
 	@mkdir -p $(PUBLIC_FOLDER)/stylesheets
-	@$(NODE_SASS) --output-style compressed -o $(PUBLIC_FOLDER)/stylesheets $(STYLESHEETS_LOC)
-	@$(POSTCSS) -r $(PUBLIC_FOLDER)/stylesheets/* --no-map
+	@$(NODE_SASS) --output-style compressed -o $(PUBLIC_FOLDER)/stylesheets $(STYLESHEETS_LOC) --source-map true
+	@$(POSTCSS) -r $(PUBLIC_FOLDER)/stylesheets/*.css --no-map
 
 # Minifies javascript files
 js:
 	@mkdir -p $(PUBLIC_FOLDER)/javascripts
-	@$(UGLIFY_JS) $(LEAFLET) $(LEAFLET_FULLSCREEN) $(JAVASCRIPTS_LOC)/*.js -m -c -o $(PUBLIC_FOLDER)/javascripts/main.js
+	@$(UGLIFY_JS) $(LEAFLET) $(LEAFLET_FULLSCREEN) $(JAVASCRIPTS_LOC)/*.js -m -c -o $(PUBLIC_FOLDER)/javascripts/main.js --source-map
+	@echo "//# sourceMappingURL=main.js.map" >> $(PUBLIC_FOLDER)/javascripts/main.js
```

Comment:
You can use `--source-map "url='<URL>'"` as an UglifyJS option here - might make it a bit neater